### DISCUSSION TITLE
Failure to download resource "x265" on MacOS 14.5

### DISCUSSION THREAD

**Discussion by mdking117** [ImageMagick/ImageMagick#7343] [Development]
### ImageMagick version

7.1.1

### Operating system, version and so on

MacOS 14.5

### Description

I'm using brew to install imagemagick and the script fails when it tries to download resource x265:
==> Fetching x265
==> Downloading https://ghcr.io/v2/homebrew/core/x265/blobs/sha256:58e2a262d8b390c5a1c61cbfb0b13e8444f9b7f238df45de743345f1cb74c950
curl: (35) Recv failure: Operation timed out                                                             #             #           #           #                                    

Error: imagemagick: Failed to download resource "x265"
Download failed: https://ghcr.io/v2/homebrew/core/x265/blobs/sha256:58e2a262d8b390c5a1c61cbfb0b13e8444f9b7f238df45de743345f1cb74c950

Any suggestions on how to resolve this?
Thanks!!

**Comment by Gounlaf** [comment]
>  curl: (35) Recv failure: Operation timed out # # # #

It's an issue from the Github network/servers/repositories/whatever there is behind ghcr.io ; re-try, one or more times. But it's not related to ImageMagick.
